lib/stripe.js

```diff
@@ -173,10 +197,32 @@ Stripe.prototype = {
       // URI-encode in case there are unusual characters in the system's uname.
       userAgent.uname = encodeURIComponent(uname) || 'UNKNOWN';
 
+      if (self._appInfo) {
+        userAgent.application = self._appInfo;
+      }
+
       cb(JSON.stringify(userAgent));
     });
   },
 
+  getAppInfoAsString() {
```

Comment:
WHAT DID I EVEN DO THERE.

Yes, that would not only work, it would also be what I meant to type.  I'm not even sure how what I typed even worked.